src/Umbraco.Web.UI.Client/src/packages/multi-url-picker/link-picker-modal/link-picker-modal.element.ts

```diff
@@ -196,9 +196,7 @@ export class UmbLinkPickerModalElement extends UmbModalBaseElement<UmbLinkPicker
 	async #getUrlForDocument(unique: string) {
 		const documentUrlRepository = new UmbDocumentUrlRepository(this);
 		const { data: documentUrlData } = await documentUrlRepository.requestItems([unique]);
-		return documentUrlData && documentUrlData[0].urls.length > 0
-			? documentUrlData?.[0].urls[0].url
-			: '';
+		return documentUrlData?.[0].urls[0].url ?? '';
```

Comment:
I'm not sure this is right @leekelleher.  I had this to start with but found an unpublished document will return an empty `urls` property.  And hence `urls[0]` would error.

Reply:
Oops! I didn't notice it error when testing it out. I'll take another look and amend/revert accordingly.